analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -644,8 +652,11 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
      * below a "target" subdirectory of the actual project.
      */
     fun parsePackage(
-        artifact: Artifact, repositories: List<RemoteRepository>,
-        localProjects: Map<String, MavenProject> = emptyMap(), sbtMode: Boolean = false
+        artifact: Artifact,
+        repositories: List<RemoteRepository>,
+        useArtifactRepositories: Boolean,
```

Comment:
Should we set this to `useArtifactRepositories = true` be default to that together with the default of `sbtMode = false` this describes the default Maven behavior?

Reply:
@oss-review-toolkit/kotlin-devs any opinion here?

Reply:
I have no strong opinion here, I'm fine with and without default value. But it would be very good to explain the argument in the function docs.